libs/npm/core/src/hooks/api-editor/admin/useApiAdminActions.ts

```diff
@@ -20,15 +20,15 @@ export const useApiAdminActions = (
 
   return React.useRef({
     findAllActions: () => {
-      return api.get<IActionModel[], AxiosResponse<IActionModel[]>, any>(`/admin/actions`);
+      return api.get<never, AxiosResponse<IActionModel[]>, any>(`/admin/actions`);
```

Comment:
what is the significance of `never` here?

Reply:
Essentially we `never` pass a value in a get body.  I find the Axios generic types annoying.